consent-ui/src/app/ais/entry-page/initiation/consent-initiate/consent-initiate.component.ts

```diff
@@ -60,7 +60,7 @@ export class ConsentInitiateComponent implements OnInit {
         this.sessionService.setConsentState(authorizationId, new AuthConsentState(res.violations));
         this.router.navigate([EntryPageAccountsComponent.ROUTE], { relativeTo: this.activatedRoute.parent });
         break;
-      case ActionEnum.INITIATE_PAYMENT:
+      case ActionEnum.SINGLE_PAYMENT:
```

Comment:
Why we specialize API and Facade-level services to `SINGLE_PAYMENT`. `INITIATE_PAYMENT` is completely fine and flows for bulk and other kinds of payments will be vey similar with the difference only in some 1-2 pages

Reply:
done